code/modules/power/supermatter/supermatter_hit_procs.dm

```diff
@@ -252,15 +252,17 @@
 	. |= FALL_STOP_INTERCEPTING | FALL_INTERCEPTED
 
 /obj/machinery/power/supermatter_crystal/proc/Consume(atom/movable/consumed_object)
+	var/object_size
 	if(isliving(consumed_object))
 		var/mob/living/consumed_mob = consumed_object
+		object_size = consumed_mob.mob_size
```

Comment:
maybe it would be a good idea to add 1 or 2 to mob size, since it starts at 0, and normal being 2 is a bit low for the other equations

Reply:
i'll add a 2 so that a body feels a bit stronger than a simple object